lib/kino/data_table.ex

```diff
@@ -240,4 +241,54 @@ defmodule Kino.DataTable do
   defp value_to_string(value) do
     if String.Chars.impl_for(value), do: "#{value}", else: "#{inspect(value)}"
   end
+
+  defp summary(%Explorer.DataFrame{} = df) do
+    describe =
+      df
+      |> Explorer.DataFrame.describe()
+      |> Explorer.DataFrame.slice([1, 3, 7])
```

Comment:
Can we switch it such that we use statistic names rather than indices, such that we don't rely on the order?